### ISSUE TITLE
proj:transform and other proj requirements?

### ISSUE THREAD

**Issue by cholmes** [planetlabs/stac-extension#4]
Looking at the readme I was expecting to see mention of proj:transform, but it didn't seem to be mentioned at all. Seems like it'd be good to include it?

I was also wondering about making it so the planet extension requires sufficient proj fields so that clients like gdal vrt, open data cube and others can get the full location information to make virtual catalogs without opening cogs. I think for that we'd need transform, shape and epsg - making them all required.

**Comment by tschaub** [comment]
I agree we can add something about the projection extension.  We'll need to have different requirements for different item types / asset types.  For example, basic assets won't have CRS info, MODIS assets are Sinusoidal (no EPSG code), etc.

**Comment by m-mohr** [comment]
A recommendation would be good, although I assume it's the same best practice/requirement as defined at https://github.com/stac-extensions/projection#best-practices , right? So maybe a link is enough? It's not Planet specific anyway...
